src/rp2040/hardware_regs/include/hardware/platform_defs.h

```diff
@@ -37,19 +37,23 @@
 
 #define PIO_INSTRUCTION_COUNT _u(32)
 
-// For USB operation this *has* to be 48 MHz
-#define USB_CLK_MHZ _u(48)
-
-// PICO_CONFIG: SYS_CLK_MHZ, The system operating frequency in MHz, type=int, default=125, advanced=true, group=hardware_base
-#ifndef SYS_CLK_MHZ
-#define SYS_CLK_MHZ _u(125)
+// PICO_CONFIG: XOSC_KHZ, The crystal oscillator frequency in kHz, type=int, default=12000, advanced=true, group=hardware_base
+// NOTE:  The system and USB clocks are generated from the frequency using two PLLs.
+// If you override this define, or SYS_CLK_KHZ below, you will *also* need to add your own adjusted PLL set-up defines to
+// overide the defaults which live in src/rp2_common/hardware_clocks/include/hardware/clocks.h
+// Please see the comments there about calculating the new PLL setting values.
+#ifndef XOSC_KHZ
```

Comment:
Given that this is a PICO_CONFIG value, renaming it from `XOSC_MHZ` to `XOSC_KHZ` feels like a backwards-incompatible change? :thinking: 
Perhaps adding
```
#ifdef XOSC_MHZ
#define XOSC_KHZ (XOSC_MHZ * KHZ)
#endif
```
is "enough" to fix that? :man_shrugging:

Reply:
**bump**

Reply:
indeed, bump